axon-server/administration/error-codes.md

```diff
@@ -0,0 +1,658 @@
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2200</td>
+      <td style="text-align:left">
+        <p><b>INVALID_TRANSACTION_TOKEN </b>
+        </p>
+        <p>This error is<b> </b>returned when during replication there is a mismatch
+          between the transaction token in the new transaction and the transaction
+          token that the event store node expects. This means that there is a likely
+          inconsistency in the data between the nodes.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2301</td>
+      <td style="text-align:left">
+        <p><b>CLUSTER_NOT_ALLOWED</b>
+        </p>
+        <p>This error is<b> </b>returned when an instruction to create a cluster
+          is rejected.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2302</td>
+      <td style="text-align:left">
+        <p><b>CONTEXT_CREATION_NOT_ALLOWED</b>
+        </p>
+        <p>This<b> </b>error is<b> </b>returned when an instruction to create a context
+          on an Axon Server cluster is rejected due to invalid permissions.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2303</td>
+      <td style="text-align:left">
+        <p><b>NOT_SUPPORTED_IN_DEVELOPMENT</b>
+        </p>
+        <p>This<b> </b>error is<b> </b>returned when an instruction valid only for
+          non-development environments is attempted on a development environment.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2304</td>
+      <td style="text-align:left">
+        <p><b>CANNOT_DELETE_INTERNAL_CONTEXT</b>
+        </p>
+        <p>This error is returned when an operation to delete any internal context
+          is attempted on an Axon Server cluster.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2305</td>
+      <td style="text-align:left">
+        <p><b>MAX_CLUSTER_SIZE_REACHED</b>
+        </p>
+        <p>This error is returned when the maximum size of the cluster is reached.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2306</td>
+      <td style="text-align:left">
+        <p><b>ALREADY_MEMBER_OF_CLUSTER</b>
+        </p>
+        <p>This error is returned when<b> </b>the maximum size of the cluster is
+          reached.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2307</td>
+      <td style="text-align:left">
+        <p><b>NOT_A_MEMBER</b>
+        </p>
+        <p>This error is returned when<b> </b>the maximum size of the cluster is
+          reached.</p>
```

Comment:
A cluster related instruction was sent to a node that is not a member of a cluster.